test/unit_tests/apps/owning-a-home/js/explore-rates/RateCheckerChart-spec.js

```diff
@@ -0,0 +1,98 @@
+const BASE_JS_PATH = '../../../../../../cfgov/unprocessed/apps/owning-a-home/';
+const RateCheckerChart = require( BASE_JS_PATH + 'js/explore-rates/RateCheckerChart' );
+
+const HTML_SNIPPET = `
+  <section id="chart-section" class="chart">
+
+    <figure class="data-enabled loading">
```

Comment:
Shouldn't the `addLoading` function add this 'loading' class?

Reply:
Good call. It should be in the markup initially, because as soon as the page loads it's "loading" and that class adds the opacity, so we don't want it to flicker on. However, I adjusted the RateCheckerChart to [call startLoading](https://github.com/cfpb/cfgov-refresh/pull/4045/commits/73339286e1b4c9e67e27c239657e017947bfc445#diff-4b12212c8ef499119732f0c7bd442999R37) on init so it'll add that class to the markup if it was forgotten.